2019-07-09-qubes-4-0-2-rc1.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Qubes OS 4.0.2-rc1 has been released!"
+categories: releases
+download_url: /downloads/
+author: Andrew David Wong
+---
+
+We're pleased to announce the first release candidate for Qubes 4.0.2!
+Features:
+
+- All 4.0 dom0 updates to date
+- Fedora 30 TemplateVM
+- Debian 10 TemplateVM
+- Whonix 15 Gateway and Workstation TemplateVMs
+- Linux kernel 4.19 by default
+
+Qubes 4.0.2-rc1 is available on the [Downloads] page.
+
+
+What is a point release?
+------------------------
+
+A point release does not designate a separate, new version of Qubes OS.
+Rather, it designates its respective major or minor release (in this
+case, 4.0) inclusive of all updates up to a certain point. Installing
+Qubes 4.0 and fully updating it results in the same system as installing
+Qubes 4.0.2.
+
+
+What should I do?
+-----------------
+
+If you installed Qubes 4.0 or 4.0.1 and have [fully updated], then
+your system is already equivalent to a Qubes 4.0.2 installation. No
+further action is required.
+
+Regardless of your current OS, if you wish to install (or reinstall)
+Qubes 4.0 for any reason, then the 4.0.2 ISO makes this more convenient
+and secure, since it bundles all Qubes 4.0 updates to date.
+
+
+Release candidate planning
+--------------------------
+
+If no major issues are discovered in 4.0.2-rc2, we expect the stable
```

Comment:
this was probably supposed to be 4.0.2-rc1

Reply:
Yes, thanks!